bigtable-dataflow-parent/bigtable-hbase-beam/src/test/java/com/google/cloud/bigtable/beam/CloudBigtableScanConfigurationTest.java

```diff
@@ -73,5 +74,72 @@ public void testToBuilder() {
     Assert.assertNotSame(config, copy);
     Assert.assertEquals(config, copy);
   }
-}
 
+  /**
+   * This ensures that the config built from regular parameters with a scan are the same as the
+   * config built from runtime parameters, so that we don't have to use runtime parameters to repeat
+   * the same tests.
+   */
+  @Test
+  public void testRegularAndRuntimeParametersAreEqualWithScan() {
+    CloudBigtableScanConfiguration withRegularParameters =
+        config.toBuilder().withConfiguration("somekey", "somevalue").build();
+    CloudBigtableScanConfiguration withRuntimeParameters =
+        new CloudBigtableScanConfiguration.Builder()
+            .withTableId(StaticValueProvider.of(TABLE))
+            .withProjectId(StaticValueProvider.of(PROJECT))
+            .withInstanceId(StaticValueProvider.of(INSTANCE))
+            .withScan(new Scan(START_ROW, STOP_ROW))
+            .withConfiguration("somekey", StaticValueProvider.of("somevalue"))
+            .build();
+    Assert.assertNotSame(withRegularParameters, withRuntimeParameters);
+    Assert.assertEquals(withRegularParameters, withRuntimeParameters);
+
+    // Verify with requests.
+    ReadRowsRequest request = withRegularParameters.getRequest();
+    withRegularParameters = withRegularParameters.toBuilder().withRequest(request).build();
+    withRuntimeParameters =
+        withRuntimeParameters.toBuilder().withRequest(StaticValueProvider.of(request)).build();
+    Assert.assertNotSame(withRegularParameters, withRuntimeParameters);
```

Comment:
Again, please remove the `assertNotSame`

Reply:
done